src/icons/Questionmark.tsx

```diff
@@ -3,8 +3,8 @@ import * as React from "react";
 
 
 const SvgQuestionmark = (props: React.SVGProps<SVGSVGElement>) => (
-  <svg width="1em" height="1em" viewBox="0 0 25 25" {...props}>
-    <g fill="pink;" fillRule="evenodd">
+  <svg width="1.3em" height="1.3em" viewBox="0 0 25 25" {...props}>
```

Comment:
Probeer de icons neutraal in het systeem te stoppen (zonder fill en met 1em width en height) zodat je de kleur kan aanpassen door een prop te zetten:
<IconQuestionmark fill={blue} />

En de size mbhv. font-size.

Check mijn prq README hoe je het component opnieuw kan genereren.